packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -36,7 +36,9 @@ export type CallCompositePage =
   | 'joinCallFailedDueToNoNetwork'
   | 'leftCall'
   | 'lobby'
-  | 'removedFromCall';
```

Comment:
I'm regretting that we added separate pages for all the end call reasons. Why do we expose to contoso exactly what the reason for ending a call was via the page? :(
This is something there should be an adapter event for. There should have been a single `callEnded` page. We could always fill in the right content based on the reason for the call being ended.

How do we feel about cutting our losses and adding a single `callEnded` page now and lumping all future call ended notices to that single page?

Reply:
https://skype.visualstudio.com/SPOOL/_workitems/edit/2976124